Application/EdFi.Ods.Common/Infrastructure/Filtering/AuthorizationFilterDefinition.cs

```diff
@@ -43,7 +42,7 @@ public class AuthorizationFilterDefinition
         public AuthorizationFilterDefinition(
             string filterName,
             string friendlyHqlConditionFormat,
-            Action<ICriteria, Junction, string, IDictionary<string, object>, JoinType> criteriaApplicator,
+            Action<ICriteria, Junction, string[], IDictionary<string, object>, JoinType, IAuthorizationStrategy> criteriaApplicator,
```

Comment:
`IAuthorizationStrategy` was added here because with the custom view-based authorization, we create them using a factory (so there will be multiple instances of the class, rather than a singleton) and each strategy instance carries additional contextual information relevant to performing authorization using the custom view.